joomla/joomla-cms
# Custom fields in contact mails are rendered with HTML

Pull Request for Issue #40338.

### Summary of Changes
This pr restores the behavior that custom fields in mails are rendered without HTML code. It was like that forever but the pr #37716 broke it.

### Testing Instructions
- Create a normal contact custom field
- Create a contact mail custom field
- Create a contact with an email address and fill in a value into the custom field
- Create a contact menu item the new contact selected
- Open the contact details page
- Send a mail to the contact via the web form

You can also do the testing instructions from #37716 to ensure that the contact data is rendered like an article.

### Actual result BEFORE applying this Pull Request
The custom field is rendered like on articles. The mail which got sent to the contact contains some HTML output.

### Expected result AFTER applying this Pull Request
The custom field is rendered like on articles. The mail which got sent to the contact contains no HTML output.

### Link to documentations
Please select:
